src/administrator/sql/install.mysql.utf8.sql

```diff
@@ -148,6 +148,7 @@ CREATE  TABLE IF NOT EXISTS `#__swa_event_ticket` (
   `name` VARCHAR(100) NOT NULL ,
   `quantity` INT NOT NULL ,
   `price` DECIMAL(6,2) NOT NULL ,
+  `notes` MEDIUMTEXT DEFAULT NULL,
```

Comment:
MEDIUMTEXT has a max size of 16,777,215, and the form limits the input to 500.
TEXT or BLOB would suffice here!